README.md

```diff
@@ -559,33 +601,52 @@ a particular load zone. This mapping can be accomplished as follows:
+demand should not be saved. `decompose_demand_profile_by_state_to_loadzone()` returns `df_lz`, which 
+is a DataFrame of the demand data mapped to each load zone.
 
 #### V. Example of Downloading and Preparing EFS Demand Data
 
 To demonstrate the work flow of these modules, this subsection presents an example of obtaining the 
 EFS demand data and subsequently preparing it for use in Breakthrough Energy Sciences' grid model. 
 In this example, EFS demand data under the 'Reference' electrification scenario and the 'Slow' 
-technology advancement scenario are acquired for the year 2030. After mapping the state-level 
-demand to the different load zones, the aggregate demand is saved as a .csv file to the current 
-working directory. This example is implemented as follows:
+technology advancement are acquired for the year 2030. After mapping the state-level demand to the 
+different load zones, the aggregate demand is saved as a .csv file to the current working directory. 
+This example is implemented as follows:
 
 ```python
 from prereise.gather.demanddata.nrel_efs.aggregate_demand import combine_efs_demand
-from prereise.gather.demanddata.nrel_efs.get_efs_data import partition_by_sector
+from prereise.gather.demanddata.nrel_efs.get_efs_data import partition_demand_by_sector
 from prereise.gather.demanddata.nrel_efs.map_states import decompose_demand_profile_by_state_to_loadzone
 
-sect_dem = partition_by_sector(es="Reference", ta="Slow", year=2030, fpath="")
+sect_dem = partition_demand_by_sector(es="Reference", ta="Slow", year=2030, fpath="")
 agg_dem = combine_efs_demand(efs_dem=sect_dem)
-agg_dem_lz = decompose_demand_profile_by_state_to_loadzone(agg_dem=agg_dem, save="EFS_Demand_Reference_Slow_2030.csv")
+agg_dem_lz = decompose_demand_profile_by_state_to_loadzone(df=agg_dem, save="EFS_Demand_Reference_Slow_2030.csv")
+```
+
+#### VI. Example of Downloading and Preparing EFS Flexibility Data
+
+This subsection presents an example of obtaining the EFS flexibility data and subsequently preparing 
+it for use in Breakthrough Energy Sciences' grid model. In this example, EFS flexibility data under 
+the 'Reference' electrification scenario, 'Slow' technology advancement, and 'Base' flexibility scenario 
+are acquired for the year 2030. The state-level sectoral flexibility is mapped to the different load 
+zones, however the different flexibility data are not saved as .csv files as was done in the prior example. 
+This example is implemented as follows:
+
+```python
+from prereise.gather.demanddata.nrel_efs.get_efs_data import partition_flexibility_by_sector
+from prereise.gather.demanddata.nrel_efs.map_states import decompose_demand_profile_by_state_to_loadzone
+
+sect_flex = partition_flexibility_by_sector(es="Reference", ta="Slow", flex="Base", year=2030, fpath="")
+sect_flex_lz = {}
+for i in sect_flex.keys():
+    sect_flex_lz[i] = decompose_demand_profile_by_state_to_loadzone(df=sect_flex[i])
 ```
```

Comment:
```
from prereise.gather.demanddata.nrel_efs.get_efs_data import partition_flexibility_by_sector
from prereise.gather.demanddata.nrel_efs.map_states import decompose_demand_profile_by_state_to_loadzone

sect_flex = partition_flexibility_by_sector(es="Reference", ta="Slow", flex="Base", year=2030, fpath="")
sect_flex_lz = {k: decompose_demand_profile_by_state_to_loadzone(df=v) for k, v in sect_flex.items()}
```

Reply:
Thanks for the suggestion! I'll make that change.

Reply:
Done.